demos/src/alternative-cookie-message.mustache

```diff
@@ -0,0 +1,2 @@
+
+<div data-o-component="o-cookie-message"></div>
```

Comment:
Could we set the alternative theme through the data attributes now? As that will work with the build service out of the box